tests/Proxy/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -62,23 +62,23 @@ pragma solidity ^0.4.24;
  */
 contract Proxy {
     /**
-    * @dev Tells the address of the implementation where every call will be delegated.
-    * @return address of the implementation to which it will be delegated
-    */
+     * @dev Tells the address of the implementation where every call will be delegated.
```

Comment:
I like this 👍

Reply:
me too. maybe I could split this PR into
1. fixing comment indentation.
2. Contract Definition edge cases